package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "hetmech-frontend",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-scripts": "2.1.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
+    "name": "hetmech-frontend",
+    "version": "0.0.1",
+    "description": "Front-end interface for Hetmec",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/greenelab/hetmech-frontend.git"
+    },
+    "author": "Vincent Rubinetti",
+    "license": "BSD-3-Clause",
+    "devDependencies": {
```

Comment:
As I said, `react-scripts` can take care of these dependencies.

Reply:
See response to above comments.